### DISCUSSION TITLE
Possible In Cpanel ??

### DISCUSSION THREAD

**Discussion by Nicat-dcw** [is-a-good-dev/register#121] [Q&A] [answered]
Hello, is it possible for me to use the domain in cpanel hosting?

**Comment by nihaddev** [comment]
"target": {
 "A": "ip adress" 
},

**Reply by Nicat-dcw** [reply]
Thanks

**Comment by wclarkey** [comment, marked as answer]
This is no longer the answer. Read our docs https://docs.is-a-good.dev
